lib/mini_profiler/profiler.rb

```diff
@@ -213,7 +213,7 @@ def tool_disabled_message(client_settings)
     def call(env)
       start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
       client_settings = ClientSettings.new(env, @storage, start)
-      MiniProfiler.deauthorize_request if @config.authorization_mode == :whitelist
+      MiniProfiler.deauthorize_request if @config.explicit_authorization_mode?
```

Comment:
I know it is ugly, but we save on one method call in a critical path... I am a bit nit picky here cause this happens on every request in production 

`if @config.authorization_mode == :allow_authorized`

Reply:
Now that you mention it, the predicate method was added primarily to wrap up the original approach for maintaining backward compatibility with clients that are currently specifying `:whitelist`.

I think the `#explicit_authorization_mode?` method is a bit unnecessary now that we've implemented your recommendation to coerce the authorization mode to `:allow_explicit` when given the old identifier.

I'm guessing you'll agree -- I think I'll just refactor it away entirely.